lib/courrier/email.rb

```diff
@@ -70,7 +70,7 @@ def self.subclass_by_email_name(email_name)
     end
 
     def recipient
-      return Settings.interceptor_emails.first if Settings.interceptor_emails.any?
+      return Courrier.configuration.interceptor_email if Courrier.configuration.interceptor_email.present?
```

Comment:
And jobs had no interceptor_emails set? Is that why you dont have to use the interceptor_email config anywhere? That would only be if we decided to use the gem in platform? THat would be a nice change btw.

Reply:
Both apps have them set to empty arrays, so this line actually doesn't do anything. It is just there to preserve the functionality if we ever choose to use it.